src/components/Attachments/AttachmentCarousel/Pager/AttachmentCarouselPagerContext.ts

```diff
@@ -3,7 +3,27 @@ import {createContext} from 'react';
 import type PagerView from 'react-native-pager-view';
 import type {SharedValue} from 'react-native-reanimated';
 
+/**
+ * The pager items array is used within the pager to render and navigate between the images
+ */
+type AttachmentCarouselPagerItems = {
+    /**
+     * The source of the image is used to identify each attachment/page in the pager
+     */
+    source: string;
+    /**
+     * The index of the pager item determines the order of the images in the pager
+     */
+    index: number;
```

Comment:
This is the standard comment style, can you please apply it to the other comments (and leave a whitespace in between properties)?

```suggestion
    /** The source of the image is used to identify each attachment/page in the pager */
    source: string;

    /** The index of the pager item determines the order of the images in the pager */
    index: number;
```